lib/theme_check/checks/translation_key_exists.rb

```diff
@@ -1,5 +1,20 @@
 # frozen_string_literal: true
 module ThemeCheck
+  module SystemTranslations
```

Comment:
Why not put this in `lib/theme_check/shopify_liquid` with the other similar files?

Reply:
Ah... I remember why. Because shopify_liquid is for liquid features. Objects, filters, etc. This kind of doesn't fit that mold. This is also why the data file is at the top of /data and not in /data/shopify_liquid. 

I definitely could extract this into its own file though. But it's not used anywhere else and would require changes to theme_check.rb as well to require it.

Reply:
Got it! Maybe `data/shopify/liquid` & `data/shopify/locales` would make sense?

Reply:
data/shopify/liquid kind of does but data/shopify/locales not really since I'm not importing the translation strings themselves, only the keys. So it'd be

```
data/
  shopify/
     liquid/
        ...
     system_translations.yml
lib/
  theme_check/
    shopify/
       liquid/
          ...
       system_translations
    ...
```


Whereas now it's
```
data/
  shopify_liquid/
    ...
  system_translation_keys.yml
lib/
  theme_check/
    shopify_liquid/
       ...
    checks/
       translation_key_exists.rb
```

Reply:
IDK feels a bit early to extract and move all those files.